src/main/java/com/negotium/negotiumapp/service/JwtUserDetailsService.java

```diff
@@ -34,6 +34,7 @@ public void setUserRepository(UserRepository userRepository) {
     @Override
     public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
         User user = userRepository.findByUsername(username);
+        System.out.println("Użytkownik " + user.getUsername() + "znaleziony");
```

Comment:
Zostałbym przy angielskim : "User " + user.getUsername() +  " has been found."

Reply:
To akurat pisałem dla siebie jak testowałem czy działa, zapomnialem skasować przed commitem